crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -62,35 +106,19 @@ pub enum NodeConfigs<T> {
         configs: Vec<NodeConfigs<T>>,
         /// Run conditions applied to everything in the tuple.
         collective_conditions: Vec<BoxedCondition>,
+        /// Metadata for `NodeConfigs`
```

Comment:
```suggestion
        /// Metadata to be applied to all elements in the tuple.
```
We can be a little more descriptive.